packages/components/src/FilterDatePicker/_docs/FilterDatePicker.stickersheet.stories.tsx

```diff
@@ -97,6 +97,7 @@ const StickerSheetTemplate: StoryFn<{ textDirection: "ltr" | "rtl" }> = ({
               inputProps={{ labelText: "Date" }}
               locale="en-AU"
               selectedDate={dateValueValidation}
+              defaultMonth={new Date("01-01-2022")}
```

Comment:
Does this work? Usually it's `yyyy-mm-dd`

Reply:
Damn it, always trips me. But yes it does haha